### PR TITLE
Implement a `open github/pullrequest` commands

### PR DESCRIPTION
**Pull request by pior**
## Why

`dad open github` and `dad open pullrequest` will be very handy.

## How

- get the remote url from the git repo using  `git config --get`
- parse the git remote url with regexp
- open the link with `xdg-open` or `open` on Mac.

See #66 

Ping @heinzf @paccorsi

### CODE REVIEW

File: pkg/cmd/open.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+	"github.com/spf13/cobra"
+
+	"github.com/pior/dad/pkg/helpers"
+	"github.com/pior/dad/pkg/project"
+)
+
+var openCmd = &cobra.Command{
+	Use:   "open [github]",
+	Short: "Open a link about your project",
```
Comment (paccorsi) [inline pkg/cmd/open.go]: "Open a link to your project" would be a better phrasing I think.

**Comment by paccorsi** [comment]
What happens if the current project doesn't have a repo associated to it ?

File: pkg/cmd/open.go (hunk 1 above)
Comment (pior) [reply pkg/cmd/open.go]: It's appropriate for the github page, but the command is supposed to open all types of links (like *Dev*, with a `open` section in dev.yml)

**Comment by pior** [comment]
It will fail, but only if you try to open the **github** page.
More links will be implemented, those not related with the git repo will not fail.

**Pull request merged**